### PR TITLE
DBZ-241 Document topic requirements

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

**Comment by jpechane** [comment]
Awestruct complained about non-ASCII chars :-(

**Comment by aviflax** [comment]
@jpechane #62 might help!

File: docs/install.asciidoc [hunk 1]
```diff
@@ -26,6 +26,20 @@ Debezium executes nightly builds and deployments into the Central snapshot repos
 
 To use a connector to produce change events for a particular source server/cluster, simply create a configuration file for the link:/docs/connectors/mysql/#configuration[MySQL Connector] or link:/docs/connectors/mongodb/#configuration[MongoDB Connector], and use the link:http://docs.confluent.io/{confluent-platform-version}/connect/userguide.html#rest-interface[Kafka Connect REST API] to add that connector configuration to your Kafka Connect cluster. When the connector starts, it will connect to the source and produce events for each inserted, updated, and deleted row or document. See the Debezium documentation for the link:/docs/connectors/mysql/[MySQL Connector] and link:/docs/connectors/mongodb/[MongoDB Connector].
 
+== Configuring Debezium Topics
+Debezium uses (either via Kafka Connect or directly) multiple topics for storing data. The topics has to be either created by an administrator or by Kafka itself by enabling auto-creation for topics. There are certain limitations and recommendations which apply to topics
```
Comment (rhauch) [inline docs/install.asciidoc]: "... topics _have_ ..."

File: docs/install.asciidoc [hunk 2]
```diff
@@ -26,6 +26,20 @@ Debezium executes nightly builds and deployments into the Central snapshot repos
...
+
+* Database history topic (for MySQL connector)
+** Infinite (or very long) retention (no compaction!)
+** Replication factor at least 3 for production
+** Single partition
+* Other topics
+** Compaction enabled
+** Replicated in production
+** Single partition
+*** You can relax the single partition rule but your application must handle out-of-order events for different rows in database (events for a single row are still totally ordered).
```
Comment (rhauch) [inline docs/install.asciidoc]: if multiple petitions are used, Kafka will determine the partition by hashing the key by default. Other partition strategies require using SMTs to set the partition number for each record.

File: docs/install.asciidoc (hunk 1 above)
Comment (jpechane) [reply docs/install.asciidoc]: Done

File: docs/install.asciidoc (hunk 2 above)
Comment (jpechane) [reply docs/install.asciidoc]: Done

**Comment by gunnarmorling** [comment]
Rebased and applied. Thanks, @jpechane!

**Pull request closed without merging**